### DISCUSSION TITLE
Repository Storage

### DISCUSSION THREAD

**Discussion by preetjhotavoni** [community/community#29667] [Other Feature Feedback, Questions, & Ideas] [answered]
What is the maximum storage available for the private repository in a free subscription?

**Comment by mazkdevf** [comment]
Hello, if i'm sure it is 500 MB or 1GB

**Comment by airtower-luna** [comment, marked as answer]
There is no hard limit, here's what the documentation says on the matter: 

> We recommend repositories remain small, ideally less than 1 GB, and less than 5 GB is strongly recommended. Smaller repositories are faster to clone and easier to work with and maintain. If your repository excessively impacts our infrastructure, you might receive an email from GitHub Support asking you to take corrective action.

For full description, see https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#repository-size-limits

**Reply by mazkdevf** [reply]
> There is no hard limit, here's what the documentation says on the matter:
> 
> > We recommend repositories remain small, ideally less than 1 GB, and less than 5 GB is strongly recommended. Smaller repositories are faster to clone and easier to work with and maintain. If your repository excessively impacts our infrastructure, you might receive an email from GitHub Support asking you to take corrective action.
> 
> For full description, see https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#repository-size-limits

👍🏻
